commit a299f68737e86f86b7f05e036c11f1f9be2e7a4f (refs/changes/32/332/1)
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2016-08-22T13:48:02-07:00 (3 years, 2 months ago)
    
    RELENG-711 builds should not use /tmp on tmpfs on ancient platforms

diff --git a/tools/jenkins-build b/tools/jenkins-build
index ac054ad..8d46b8b 100755
--- a/tools/jenkins-build
+++ b/tools/jenkins-build
@@ -31,6 +31,16 @@
 set -o errexit
 unset LD_LIBRARY_PATH   # ensure don't get Java's libs (see OS-703)
 
+#
+# Refuse to build if /tmp is tmpfs until OS-5608 is fixed.
+# Try to umount if mounted, if that fails: blow up.
+#
+if mount | grep ^/tmp; then
+    if ! umount /tmp; then
+        echo "FATAL: refusing to use mounted /tmp (See: RELENG-711)" >&2
+        exit 1
+    fi
+fi
 
 # ---- First pass:
 # 1. parse params
